app/controllers/application_pets_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ApplicationPetsController < ApplicationController
+  def update
+    application_pet = ApplicationPet.find_by(application_id: params[:application_id], pet_id: params[:pet_id])
+    application_pet.update(approved: true)
+    application_pet.pet.update(status: "Pending")
```

Comment:
- sets that record's `approved` value to true (this will help the pet know which application it was approved for later on)
- updates that pet's status